src/dxss/solve_3d.py

```diff
@@ -160,84 +156,86 @@ def dt_sample_sol(t, xu):
     )
 
 
-st = space_time(
+ST = SpaceTime(
     q=q,
     qstar=qstar,
     k=k,
     kstar=kstar,
     N=N,
     T=T,
     t=t0,
-    msh=msh,
-    Omega_Ind=omega_Ind,
-    stabs=stabs,
+    msh=MSH,
+    Omega_Ind=omega_ind,
```

Comment:
```suggestion
    omega_ind=omega_ind,
```
It looks like this has been changed to lower case in changes to `SpaceTime` definition

Reply:
😱 how did the tests/linter allow this?
I thought I renamed the symbol everywhere.

- [ ] check that this hasn't happened anywhere else.